lib/jsdom/living/nodes/HTMLLabelElement-impl.js

```diff
@@ -2,31 +2,16 @@
 
 const HTMLElementImpl = require("./HTMLElement-impl").implementation;
 const MouseEvent = require("../generated/MouseEvent");
-const { closest } = require("../helpers/traversal");
 const { domSymbolTree } = require("../helpers/internal-constants");
-const { isDisabled } = require("../helpers/form-controls");
+const NODE_TYPE = require("../node-type");
+const { isLabelable, isDisabled } = require("../helpers/form-controls");
 
-function isLabelable(node) {
-  // labelable logic defined at: https://html.spec.whatwg.org/multipage/forms.html#category-label
-  if (node.nodeType !== 1) {
-    return false;
+function getRootNode(node) {
```

Comment:
You can use the same function in traversal.js.

Reply:
I knew i missed something there.